develop/parachains/intro-polkadot-sdk.md

```diff
@@ -54,6 +54,32 @@ Every blockchain platform relies on a decentralized network of computers—calle
 
 FRAME provides the core modular and extensible components that make the Substrate SDK flexible and adaptable to different use cases. FRAME includes Rust-based libraries that simplify the development of application-specific logic. Most of the functionality that FRAME provides takes the form of plug-in modules called [pallets](/polkadot-protocol/glossary#pallet){target=\_blank} that you can add and configure to suit your requirements.
 
+``` mermaid
+graph TB
+    subgraph sg1[Client/Host]
```

Comment:
please use this as an oppurtunity and:

1. Add somewhere in glossary that in substrate, "Client", "Node" and "Host" are the same.
2. For consistency, we always prefer to say "Node", so update all instances to use this keyword foremost.

Reply:
I've opened a ticket to address this feedback. Thank you for the clarification!